src/main/java/io/github/railroad/project/ui/project/newProject/details/ForgeProjectCreationPane.java

```diff
@@ -28,6 +42,84 @@ public class ForgeProjectCreationPane extends BorderPane {
+                updateProgress(5, 10);
+                Thread.sleep(500);
+                manager.NewProject(new Project(projectPath, this.data.projectName()));
+                updateProgress(6, 10);
+                Thread.sleep(500);
+                File gradlePropertiesFile = projectPath.resolve("gradle.properties").toFile();
+
+                updateKeyValuePairByLine("mod_id", this.data.modId(), gradlePropertiesFile);
+                updateKeyValuePairByLine("mod_name", this.data.modName(), gradlePropertiesFile);
+                updateKeyValuePairByLine("mod_version", this.data.version(), gradlePropertiesFile);
+                updateKeyValuePairByLine("mod_group_id", this.data.groupId(), gradlePropertiesFile);
+                updateKeyValuePairByLine("mod_authors", this.data.author().orElse(""), gradlePropertiesFile);
+                updateKeyValuePairByLine("mod_description", this.data.description().orElse(""), gradlePropertiesFile);
+                System.out.println("gradle.properties updated successfully.");
+                updateProgress(7,10);
+                Thread.sleep(500);
+
+                File com = projectPath
+                        .resolve("src")
+                        .resolve("main")
+                        .resolve("java")
+                        .resolve("com")
+                        .resolve(this.data.author().orElse(this.data.projectName()))
+                        .resolve(this.data.modId()).toFile();
+                com.mkdirs();
+                Files.copy(projectPath
+                                .resolve("src")
+                                .resolve("main")
+                                .resolve("java")
+                                .resolve("com")
+                                .resolve("example")
+                                .resolve("examplemod")
+                                .resolve("Config.java").toAbsolutePath(),
+                        com.toPath().resolve("Config.java").toAbsolutePath());
+                updateProgress(8,10);
+                Thread.sleep(500);
+                Files.copy(projectPath
+                                .resolve("src")
+                                .resolve("main")
+                                .resolve("java")
+                                .resolve("com")
+                                .resolve("example")
+                                .resolve("examplemod")
+                                .resolve("ExampleMod.java").toAbsolutePath(),
+                        com.toPath().resolve(this.data.modId()+".java").toAbsolutePath());
+                updateProgress(9,10);
+                Thread.sleep(500);
+                updateProgress(10,10);
+            } catch (IOException e) {
+                // Handle errors
+                Platform.runLater(() -> showErrorAlert("Error", "An error occurred while creating the project.", e.getMessage()));
+            }
+            return null;
+
+        }
+    }
```

Comment:
This is *okay* as a temporary solution, but ideally we do have this split up into some proper system later down the line.